components/squad/wikis/trackmania/squad_custom.lua

```diff
@@ -0,0 +1,62 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:Squad/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Squad = require('Module:Squad')
+local SquadRow = require('Module:Squad/Row')
+local Json = require('Module:Json')
+local Variables = require('Module:Variables')
+local ReferenceCleaner = require('Module:ReferenceCleaner')
+
+local CustomSquad = {}
+
+function CustomSquad.run(frame)
+	local squad = Squad()
+	squad:init(frame):title():header()
+
+	local args = squad.args
+
+	local index = 1
+	while args['p' .. index] ~= nil or args[index] do
+		local player = Json.parseIfString(args['p' .. index] or args[index])
+		local row = SquadRow(frame, player.role, {useTemplatesForSpecialTeams = true})
```

Comment:
Just checking that `useTemplatesForSpecialTeams` is wanted. It's use is having `|newteam=Retired` instead of `|newteamrole=Retired`. Different wiki's do it different and just want to ensure that this is correct for Trackmania

Reply:
As what I found the wiki uses `|newteam=Retired`. But there may be some inconsistencies across the wiki